spec.bs

```diff
@@ -3193,6 +3188,20 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=reporting context map=]
 1. Let |response| be the [=server auction response=] which is the result of
    deserializing |auctionConfig|'s [=auction config/server response=]
    with |requestContext| according to [Section 2.3.5 of the Bidding and Auction Services IETF standard](https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#name-parsing-a-response).
+1. If |response|'s [=server auction response/nonce=] [=map/exists=]:
+  1. Let |capturedAuctionHeaders| be |global|'s [=associated Document's=] [=node navigable's=]

```

Comment:
probably `|capturedAuctionNonceHeaders|` to make it clearer, since we had some other headers named |capturedAuctionHeaders| (which I think would be best to rename to `|capturedAuctionResultHeaders|` and `|capturedAuctionSignalsHeaders|` as well)

Reply:
Did you commit? I don't see a new commit or any change.